src/JsonLd/ContextBuilder.php

```diff
@@ -145,6 +145,15 @@ private function getResourceContextWithShortname(string $resourceClass, int $ref
         $context = $this->getBaseContext($referenceType);
         $propertyContext = $operation ? ['normalization_groups' => $operation->getNormalizationContext()['groups'] ?? null, 'denormalization_groups' => $operation->getDenormalizationContext()['groups'] ?? null] : ['normalization_groups' => [], 'denormalization_groups' => []];
 
+        if($operation->getRemoveJsonLdIdType()) {
```

Comment:
T
```suggestion
        if($operation->getAliasIdType()) {
```

Nitpicking but technically they are. Or removed but just mapped / aliased using the context.